ang/test/civiawards/award-creation/directives/review-panels/review-panels.directive.spec.js

```diff
@@ -322,6 +323,161 @@
       });
     });
 
+    describe('when deleting an existing review panel', () => {
+      let crmConfirmOnFn, crmConfirmCallbackFn;
+
+      beforeEach(() => {
+        crmConfirmOnFn = jasmine.createSpy('crmConfirmOnFn').and.callFake(function (evtName, crmConfirmCallbackFntn) {
+          crmConfirmCallbackFn = crmConfirmCallbackFntn;
+        });
+
+        CRM.confirm.and.returnValue({ on: crmConfirmOnFn });
+      });
+
+      beforeEach(() => {
+        $scope.awardId = '10';
+        createController();
+        $scope.$digest();
+
+        $scope.handleDeleteReviewPanel($scope.existingReviewPanels[0]);
+      });
+
+      it('confirms before deleting the review panel', () => {
+        expect(CRM.confirm).toHaveBeenCalledWith({
+          title: 'Delete New Panel?',
+          message: 'Are you sure you want to delete this?'
+        });
+      });
+
+      describe('when yes is pressed on confirmation prompt', () => {
+        beforeEach(() => {
+          crmConfirmCallbackFn();
+          $scope.$digest();
+        });
+
+        it('deletes the review panel if yes is pressed', () => {
```

Comment:
`it('deletes the review panel', () => {`

No need to add the condition in the expectation.